keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/api/ImageSpec.kt

```diff
@@ -12,7 +14,14 @@ data class ImageSpec(
   val storeType: StoreType,
   val artifactStatuses: Set<ArtifactStatus> = enumValues<ArtifactStatus>().toSet(),
   override val application: String // the application an image is baked in
-) : ResourceSpec {
+) : ResourceSpec, UnhappyControl {
   @JsonIgnore
   override val id: String = artifactName
+
+  @JsonIgnore
+  override val maxDiffCount = 2
```

Comment:
Does this mean it will retry baking twice if it fails?

Reply:
Yes it does I think. I just stole whatever number we were using for clusters w/o digging too much into it

Reply:
It's the number of attempts in total, so 1 retry.

Reply:
That seems to conflict with what the PR says which is that there's no point retrying. Shouldn't this be `1`?

Reply:
Ehhh, maybe i'll leave it at 2 so it retries once. I can always bump it down.